src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/GameReports/PivotTeamStandingsViewModel.cs

```diff
@@ -26,18 +22,19 @@ public class PivotTeamStandingsViewModel : TeamStandingsViewModelBase
         /// <summary>
         /// Gets the sets ratio of the team.
         /// </summary>
+#pragma warning disable S1244 // Want to check floating point not with exact values, but with range
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)

Reply:
No need for this suppression here

Reply:
ok